rust/definitions/src/helpers.rs

```diff
@@ -54,11 +58,46 @@ pub fn multisigner_to_encryption(m: &MultiSigner) -> Encryption {
     }
 }
 
+pub enum IdenticonStyle {
+    /// Default style for substrate-based networks, dots in a circle.
+    Dots,
+
+    /// Blockies style used in Ethereum networks.
+    Blockies,
+}
+
 /// Print identicon from
 /// [`MultiSigner`](https://docs.rs/sp-runtime/6.0.0/sp_runtime/enum.MultiSigner.html)  
 #[cfg(feature = "signer")]
-pub fn make_identicon_from_multisigner(multisigner: &MultiSigner) -> Vec<u8> {
-    make_identicon(&multisigner_to_public(multisigner))
+pub fn make_identicon_from_multisigner(
+    multisigner: &MultiSigner,
+    style: IdenticonStyle,
+) -> Vec<u8> {
+    match style {
+        IdenticonStyle::Dots => make_identicon(&multisigner_to_public(multisigner)),
+        IdenticonStyle::Blockies => {
+            if let MultiSigner::Ecdsa(ref public) = multisigner {
+                use eth_blockies::{eth_blockies_png_data, SeedString};
+                let account = print_ethereum_address(public).unwrap();
+                let account = account.canonicalize_ethaddr();
+                let dimension = (72, 72);
+                let compressed_output = false;
+                eth_blockies_png_data(account, dimension, compressed_output)
+            } else {
+                EMPTY_PNG.to_vec()
+            }
+        }
+    }
+}
+
+#[cfg(feature = "signer")]
+pub fn make_identicon_from_id20(id: &[u8; 20]) -> Vec<u8> {
+    use eth_blockies::eth_blockies_png_data;
+
+    let account = format!("0x{}", hex::encode(&id));
+    let dimension = (72, 72);
```

Comment:
Should we keep dimension somewhere outside of these to functions as to have single source of truth if we want to modify this later on and not need to change value in multiple places?